common/src/main/java/com/mx/path/core/common/accessor/UpstreamSystemUnavailable.java

```diff
@@ -34,6 +34,18 @@
  * <p>See {@link PathRequestException} for usage details
  */
 public class UpstreamSystemUnavailable extends AccessorSystemException {
+  public UpstreamSystemUnavailable(String message) {
+    super(message);
+    setReport(false);
+    setStatus(PathResponseStatus.UPSTREAM_SERVICE_UNAVAILABLE);
+  }
+
+  public UpstreamSystemUnavailable(String message, Throwable cause) {
+    super(message, cause);
+    setReport(false);
+    setStatus(PathResponseStatus.UPSTREAM_SERVICE_UNAVAILABLE);
+  }
+
   public UpstreamSystemUnavailable(String message, String userMessage) {
```

Comment:
I think deprecating the constructors with `userMessage` makes sense here since this is an `AccessorSystemException` and we probably don't need to explain to the end user why the upstream system is unavailable.

Reply:
I agree with this. I think it should be deprecated

Reply:
Fixed.